install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -109,6 +109,7 @@ rules:
   - ciliumclusterwidenetworkpolicies/status
   verbs:
   # Update the auto-generated CNPs and CCNPs status.
+  - patch
```

Comment:
This commit needs to be backported to all branches because the commit that introduced the regression was also made available to all branches.